includes/export/class-evf-entry-csv-exporter.php

```diff
@@ -145,11 +147,11 @@ protected function generate_row_data( $entry ) {
 
 			} elseif ( is_callable( array( $this, "get_column_value_{$column_id}" ) ) ) {
 				// Handle special columns which don't map 1:1 to entry data.
-				$value = $this->{"get_column_value_{$column_id}"}( $entry );
+				$value     = $this->{"get_column_value_{$column_id}"}( $entry );
+				$raw_value = $this->{"get_column_value_{$column_id}"}( $entry );
 			}
 
-			// Ref: https://stackoverflow.com/questions/20444042/wordpress-how-to-sanitize-multi-line-text-from-a-textarea-without-losing-line .
-			$row[ $column_id ] = implode( "\n", array_map( 'sanitize_textarea_field', explode( "\n", $value ) ) );
+			$row[ $column_id ] = apply_filters( 'everest_forms_format_csv_field_data', sanitize_text_field( $value ), $raw_value, $column_id, $column_name, $columns, $entry );
```

Comment:
@krishna-suwal I think we can use like this too because I don't see any difference in `value` and `raw_value` in this filter:
`apply_filters( 'everest_forms_format_csv_field_data', sanitize_text_field( $value ), $value, $column_id, $column_name, $columns, $entry );`

Reply:
@shivapoudel the `value` has been passed through a hook so it won't maintain the initial value (please see the image below).

![image](https://user-images.githubusercontent.com/58679177/81491462-8a458c80-92ae-11ea-8d5d-66866d3d40bc.png)

Reply:
See https://github.com/wpeverest/everest-forms/pull/436/commits/6b9375884b792739268aa3543326095bcb024b97